SaraAlert/SaraAlert
# SARAALERTT-591: Support Arabic

# Description
Jira Ticket: [SARAALERT-591](https://tracker.codev.mitre.org/browse/SARAALERT-591)

Translations [here](https://teams.microsoft.com/l/file/3E2821BD-623F-451E-B51C-E8CF48FA73A5?tenantId=c620dc48-1d50-4952-8b39-df4d54d74d82&fileType=docx&objectUrl=https%3A%2F%2Fmitre.sharepoint.com%2Fsites%2FCoVDT%2FShared%20Documents%2FGeneral%2FProduct%20Owner%20Materials%2FTranslations%2FTranslation%20Feedback%2FArabic%20Right%20to%20Left.docx&baseUrl=https%3A%2F%2Fmitre.sharepoint.com%2Fsites%2FCoVDT&serviceName=teams&threadId=19:00d3d45c64ca457795d58c2764d517cd@thread.skype&groupId=7de0f318-3170-4f1c-82eb-95bfe132bfe9)

- Enable Arabic
- Support right to left displaying of text

![Screen Shot 2021-10-08 at 2 30 22 PM](https://user-images.githubusercontent.com/9956561/136609038-635f8d4c-fd07-4ddb-ba5f-1a7f2da22a13.png)
![Screen Shot 2021-10-08 at 2 30 47 PM](https://user-images.githubusercontent.com/9956561/136609040-788ada77-d97d-4823-9c77-16245b8da6a2.png)
![Screen Shot 2021-10-08 at 2 31 45 PM](https://user-images.githubusercontent.com/9956561/136609041-effc185e-dfdd-46a5-989a-e53c50a0b384.png)
![Screen Shot 2021-10-08 at 2 32 37 PM](https://user-images.githubusercontent.com/9956561/136609043-077797dd-2167-4e1d-979f-655df265abf1.png)
![Screen Shot 2021-10-08 at 2 32 55 PM](https://user-images.githubusercontent.com/9956561/136609044-c2a5111b-ec9f-4b25-bd6e-6d6433d150b5.png)

## Important Changes
Please list important files (meaning substantial or integral to the PR) along with a list of the general changes that should be highlighted for reviewers.

`config/locales/*.yml`
- added direction (`rtl` for arabic and `ltr` for everything else)

`app/views/patient_mailer/*.html.erb`, `app/javascript/components/patient/assessment/steps/*.js`
- included direction for text to be displayed

## Testing Recommendations
1. Enroll new patient with language set to Arabic
2. Set contact method to email
    i. ```PatientMailer.enrollment_email(Patient.last).deliver_now```
    ii. ```PatientMailer.assessment_email(Patient.last).deliver_now``` (might need to comment out last reminder sent logic)
    iii. ```PatientMailer.closed_email(Patient.last).deliver_now```
3. Click on weblink from enrollment or assessment email and verify translations on webpage
4. Make sure other languages still work properly since directions were added

# Checklists

**Submitter:**


@tstrass :


@ngfreiter :


@Bialogs :
